client/src/lib/components/sandbox/arena/ArenaCanvasMeleePhase.svelte

```diff
@@ -205,6 +207,7 @@
     Object.values(orders).flat().forEach((meleeOrder: GamePieceOrder) => {
       if (meleeOrder.meleeAttack) meleeAttackActions.push(meleeOrder.meleeAttack);
     });
+    isLoading = true;
```

Comment:
Is there a problem that we never set this back to false?  e.g. the phase is submitted and rejected for some reason, or the request times out.  It's still the current phase, but we're still "loading"

Reply:
I originally had this setting `isLoaded = false` after the submit phase operation completes, then realized that it wasn't needed because the next thing we do is call `rerender()` and show the new phase state, which is a new component which itself initializes `isLoaded` to false again.

It is definitely possible there's some failure edge case I'm not thinking about though.

If the server were to reject the mutation, I believe this would raise an exception here, and technically as-is this would just log that error to the console when really we probably want some better user-understandable error handling.

Either way, I think that can probably be handled by another PR focused on edge case error handling.